modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -148,11 +150,12 @@ class GAPI_EXPORTS GInferInputs
  * @{
  * @brief G-API object used to collect network outputs
  */
-struct GAPI_EXPORTS GInferOutputs
+struct GAPI_EXPORTS_W_SIMPLE GInferOutputs
```

Comment:
Need to avoid `GAPI_EXPORTS_W_SIMPLE`, I've filed a ticket, but let's to discuss it locally

Reply:
Fixed,  moved class statement to `shared_ptr`